test/scalecube-transport/providers.test.js

```diff
@@ -0,0 +1,396 @@
+import { Transport } from '../../src/scalecube-transport/Transport';
```

Comment:
test/scalecube-transport/providers.test.js --> test/scalecube-transport/Transport.test.js

Reply:
I see the logic - tests that should both for every provider are basically the tests for Transport itself.